MSAL/test/integration/native_auth/end_to_end/sign_up/MSALNativeAuthSignUpUsernameEndToEndTests.swift

```diff
@@ -131,7 +134,12 @@ final class MSALNativeAuthSignUpUsernameEndToEndTests: MSALNativeAuthEndToEndBas
         let submitCodeExp = expectation(description: "submit code")
```

Comment:
In the line 127:

`sut.signUp(username: usernameOTP, attributes: attributes, correlationId: correlationId, delegate: signUpStartDelegate)`

The global variable `attributes = ["age": 40]` is being sent, but it shouldn't since this test checks `...withEmailVerificationAsFirstStepAndCustomAttributes`. The attributes that are mandatory are sent in the line 153.

Reply:
Done.